tests/test_formatutils.py

```diff
@@ -48,6 +52,40 @@ def test_get_fstr_args():
     return results
 
 
+@pytest.mark.parametrize(('sample', 'expected'), zip(_TEST_TMPLS, [
+        ([], [('hello', str)]),
+        ([], [('hello', str)]),
+        ([], [('hello', str), ('width', str)]),
+        ([], [('hello', str), ('fchar', str), ('width', str)]),
+        ([(0, str), (1, int), (2, float)], []),
+        # example 6 is skipped
+]))
+def test_get_format_args(sample, expected):
+    """Test `get_format_args` result as tuples."""
+    assert get_format_args(sample) == expected
+
+
+@pytest.mark.parametrize(('sample', 'expected'), zip(_TEST_TMPLS, [
```

Comment:
hehe this inline parametrize is kind of long, just curious, is this common practice with pytest?